packages/web/test/src/lib/myalgowallet-mode.ts

```diff
@@ -87,5 +106,22 @@ describe("Webmode - MyAlgo Wallet test cases ", () => {
 				await connector.makeAndSignTx([execParams], txnParams);
 			});
 		});
+
+		it("Should sign a lsig transaction and return a SignedTransaction object", async function () {
+			const execParams: types.AlgoTransferParam = {
+				type: types.TransactionType.TransferAlgo,
+				sign: types.SignType.LogicSignature,
+				fromAccountAddr: lsigAccount.address(),
+				lsig: lsigAccount,
+				toAccountAddr: receiver.addr,
+				amountMicroAlgos: 1e6,
+				payFlags: {},
+			};
+			const txnParams = await getSuggestedParams(algodClient);
+			const transactions: Transaction[] = connector.makeTx([execParams], txnParams);
+			assert.doesNotThrow(() => {
+				connector.signLogicSignatureTxn(transactions[0], lsigAccount);
+			});
```

Comment:
the description say it returns SignedTransaction but we only check if it doesn't throw an error